src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -174,6 +174,11 @@ export interface IContextualMenuItem {
   onMouseDown?: (item: IContextualMenuItem, event: any) => void;
 
   /**
+   * Optional identifier for automated tests to find the element by.
+   */
+  automationId?: string;
```

Comment:
instead it'd be ideal to just extend React.HTMLProps<HTMLButtonElement> and then mix in button/anchor props in. Make it work like button.

I have a separate PR that changes contextualmenuitems into just regular Buttons, so making automationId work here in a different way would mean headache downstream.